run-in-retro-term.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+#
+# cool-retro-term support for tetris.sh
+# Author: Rojen Zaman <rojen@riseup.net>
+# License: WTFPL
+
+stdout="/dev/null"                                                                      # default stdout, optional verbose mode is /dev/stdout
+script_dir=$(dirname $(readlink -f $0))                                                 # script dir, it is required for cool-retro-term
+pl=${VARIABLE:-bash}                                                                    # if pl not specified return bash
+hl=${VARIABLE:-english}                                                                 # default language for bash
+console_profile=${VARIABLE:-Monochrome Green}                                           # default console profile
+available_languages=$(ls -I README.md $script_dir/lang/ 2>/dev/null | cut -d. -f1)      # https://github.com/rojen/tetris/blob/master/lang/README.md
+
+usage_message_0() {
+printf "%s\n" \
+    "" \
+    "usage : $0 [-h] [-v] [-s PL] [-c console]" \
+    "" \
+    " -h     help" \
+    " -v     verbose"
+}
+
+usage_message_1() {
+printf "%s\n" \
+    "" \
+    " -s     select programming language of tetris" \
+    "    c" \
+    "    java" \
+    "    javascript" \
+    "    perl" \
+    "    python" \
+    "    ruby" \
+    "" \
+    "    bash [-l language]  select human language of tetris (only work with bash):"
+}
+
+usage_message_2() {
+printf "%s\n" \
+    "" \
+    "english" \
+    "$available_languages" \
+    ""
+}
+
+usage_message_3() {
+printf "%s\n" \
+    "" \
+    " -c    console profile" \
+    "    1      Default Amber" \
+    "    2      Monochrome Green" \
+    "    3      Green Scanlines" \
+    "    4      Default Pixelated" \
+    "    5      Apple ][" \
+    "    6      Vintage" \
+    "    7      IBM Dos" \
+    "    8      IBM 3278" \
+    "    9      Futuristic"
+}
+
+# usage
+usage() {
+    usage_message_0
```

Comment:
Hmmm, why do you think we may need to split printing out usage like this?

Reply:
Ah, sorry, now I understand the logic, let's rename functions to make names reflect what part of the message we print.